sdk/synapse/Azure.Analytics.Synapse.Artifacts/tests/DatasetClientLiveTests.cs

```diff
@@ -39,7 +40,8 @@ await foreach (var expectedDataset in DatasetClient.GetDatasetsByWorkspaceAsync(
         [Test]
         public async Task TestCreateDataset()
         {
-            var operation = await DatasetClient.StartCreateOrUpdateDatasetAsync("MyDataset", new DatasetResource(new Dataset(new LinkedServiceReference(LinkedServiceReferenceType.LinkedServiceReference, "testsynapseworkspace-WorkspaceDefaultStorage"))));
+            string workspaceName = TestEnvironment.WorkspaceUrl.Split('.')[0].Split('/')[2];
```

Comment:
Since you seem to do this more than once, perhaps add a property or method to your `TestEnvironment` that just returns this. Would make the call sites more concise and make it easier to change the parsing in a single spot.